apps/files/src/components/LocationPicker/LocationPicker.vue

```diff
@@ -9,14 +9,20 @@
       <oc-button @click.native="leaveLocationPicker(originalLocation)">
         <translate>Cancel</translate>
       </oc-button>
-      <oc-button
-        id="location-picker-btn-confirm"
-        variation="primary"
-        :disabled="!canConfirm"
-        @click.native="confirmAction"
-      >
-        <span v-text="confirmBtnText" />
-      </oc-button>
+      <span v-if="!canConfirm" :uk-tooltip="sameLocationToolTip">
```

Comment:
Add pls `key` to `if` `else`. There might be an issue with rendering without it https://vuejs.org/v2/style-guide/#v-if-v-else-if-v-else-without-key-use-with-caution